ledger/acctupdates.go

```diff
@@ -1367,6 +1367,40 @@ func (au *accountUpdates) lookupResource(rnd basics.Round, addr basics.Address,
 	}
 }
 
+func (au *accountUpdates) lookupAccountDeltas(rnd basics.Round) (ledgercore.AccountDeltas, error) {
```

Comment:
@tzaffi Look at the diff between this function and https://github.com/algorand/go-algorand/blob/master/ledger/acctupdates.go#L895 w/r/t what we discussed today.